README.md

```diff
@@ -1,6 +1,22 @@
-# Steamship Python Client Library
+<img src="docs/_static/Steamship-symbol-dark.png" alt="Steamship Logo" width="100"/>
 
-Steamship is the fastest way to build, ship, and use full-lifecycle language AI.
+# Steamship Python SDK
+
+[Steamship](https://steamship.com) is the fastest way to build, ship, and use full-lifecycle AI. 🚢 
+
+This library is for:
```

Comment:
do we want to mention agents here and above?  it might help segue into the next question.

Reply:
Better?